bin/setup

```diff
@@ -26,7 +26,7 @@ FileUtils.chdir APP_ROOT do
   # end
 
   puts "\n== Preparing database =="
-  system! 'bin/rails db:prepare'
+  system! 'bin/rails db:drop db:prepare'
```

Comment:
I love that you updated the setup scripts too. That's great attention to detail

Reply:
Thanks 😊